public/_includes/_hero-home.jade

```diff
@@ -6,8 +6,8 @@ header(class="background-sky l-relative")
         a(href="/docs/ts/latest/quickstart.html" class="hero-cta md-raised button button-large button-plain" md-button) Get Started
 
 
-    announcement-bar
-      .announcement-bar-slide.clearfix
+    announcement-bar.announcement-bar
```

Comment:
Given that the styles have been changed. Could we just drop the `.announcement-bar` and adjust the styles to apply to the `announcement-bar` tag?